velox/type/Type.cpp

```diff
@@ -57,6 +57,7 @@ TypeKind mapNameToTypeKind(const std::string& name) {
       {"VARBINARY", TypeKind::VARBINARY},
       {"TIMESTAMP", TypeKind::TIMESTAMP},
       {"DATE", TypeKind::DATE},
+      {"DECIMAL", TypeKind::SHORT_DECIMAL},
```

Comment:
I find this confusing. Shouldn't we map "SHORT_DECIMAL" to TypeKind::SHORT_DECIMAL and "LONG_DECIMAL" to TypeKind::LONG_DECIMAL.

Also, what's the motivation to use a single DecimalType class instead of two: ShortDecimalType and LongDecimalType?

Reply:
We could create 2 decimal types and strings. But how would our function signatures look like?